src/pages/events/icbd3/icbd-page-view.njk

```diff
@@ -180,21 +180,20 @@
                             </div>
                         </div>
 
-                        <div id="mini-talk" class="event">
+                        <div id="mini-talks" class="event">
                             <div class="front">
                                 <h3>
                                     <i class="fas fa-chalkboard-teacher"></i>
-                                    Specialization talks</h3>
+                                    Mini talks</h3>
                             </div>
 
                             <div class="details">
                                 <h4>What does a career in your dream field look like?</h4>
                                 <p>
-                                    Data Science, Cyber Security, Software Engineering… These are all broad IT fields you have heard about but perhaps you would 
-                                    like to have a more concrete understanding of what the daily life of an engineer in such domains is like. During this activity, 
+                                    Data Science, Cyber Security, Software Engineering… These are all broad IT fields <a href="https://t.me/clicnews/382">you have let us know you are the most interested by</a>! Therefore, these mini-talks are there to give you a more concrete understanding of what the daily life of an engineer in such domains is like. During these activities, 
                                     several alumni working in the same field will come to give a quick overview of a project they worked on before discussing their 
                                     daily routine and give you insights on what your future work life may look like. We will host alumni working in domains such 
```

Comment:
insights to => insights into

Reply:
Remplacer insights **on** par insights **into** tu veux dire ?

Reply:
ah yes c'est ca que je voulais dire, my bad😅